specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2018-02-01-preview/containerregistry_build.json

```diff
@@ -0,0 +1,2220 @@
+      "description": "The parameters for updating a build definition.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/BuildDefinitionPropertiesUpdateParameters",
+          "description": "The properties for updating a build definition.",
+          "x-ms-client-flatten": true
+        },
+        "tags": {
+          "description": "The ARM resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "BuildDefinitionPropertiesUpdateParameters": {
+      "description": "The properties for updating a build definition.",
+      "type": "object",
+      "properties": {
+        "alias": {
+          "description": "The alternative updatable name for a build definition.",
+          "type": "string"
+        },
+        "status": {
+          "description": "The current status of build definition.",
+          "enum": [
+            "Disabled",
+            "Enabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "BuildDefinitionStatus",
+            "modelAsString": true
+          }
+        },
+        "timeout": {
+          "format": "int32",
+          "description": "Build Time out in seconds.",
+          "type": "integer"
+        },
+        "isBaseImageTriggersEnabled": {
+          "description": "The value of this property indicates whether auto triggers of the build items trigger a build against this definition.",
+          "default": false,
+          "type": "boolean"
+        },
+        "sourceRepository": {
+          "$ref": "#/definitions/SourceRepositoryUpdateParameters",
+          "description": "The properties that describes the source(code) for the build definition."
+        }
+      }
+    },
+    "SourceRepositoryUpdateParameters": {
+      "description": "The properties for updating the source code repository configuration.",
+      "type": "object",
+      "properties": {
+        "sourceControlAuthProperties": {
+          "$ref": "#/definitions/SourceControlAuthInfo",
+          "description": "The authorization properties for accessing the source code repository."
+        },
+        "isCommitTriggerEnabled": {
+          "description": "The value of this property indicates whether the source control commit trigger is enabled or not.",
+          "default": false,
+          "type": "boolean"
+        }
+      }
+    },
+    "Build": {
+      "description": "The properties for a build.",
+      "type": "object",
+      "properties": {
+        "buildId": {
+          "description": "The unique identifier for the build.",
+          "type": "string"
+        },
+        "status": {
+          "description": "The current status of the build.",
+          "enum": [
+            "Queued",
+            "Started",
+            "Running",
+            "Succeeded",
+            "Failed",
+            "Cancelled"
```

Comment:
`Canceled`

Reply:
ARM had that spelling for its provisioning state. Let us have correct state as this is Build state

Reply:
I believe `canceled` is the correct spelling for American English but I am not a language expert either :-) I wasn't sure if we want to confuse people with two different spellings within one service.